packages/leancode_debug_page/README.md

```diff
@@ -104,3 +91,14 @@ You can configure debug page's entry points by setting ```showEntryButton``` (de
 ## Warning
 
 For gathering logs from loggers, this package relies on listening to `Logger.root`. This means that changing `Logger.root.level` affects this package's behavior, and the logs are only collected from the current isolate.
+
+---
+
+<p style="text-align: center;">
+   <a href="https://leancode.co/?utm_source=readme&utm_medium=bloc_lens_package">
```

Comment:
Replace `bloc_lens_package` with `leancode_debug_page` here and below

Reply:
🙈